intro-HPC/intro-HPC.tex

```diff
@@ -83,6 +83,10 @@ \part{Advanced Guide}
 
 \include{ch_best_practices}
 
+\part{Software-specific Best Practices}
+
+\include{ch_MATLAB}
```

Comment:
@redfast00 Wrap this under `\ifgent` for now, I'll need to check with the other sites which parts of the `MATLAB` chapter are relevant to them too.

Let's get this merged first for Ghent, and then open an issue where other sites can pitch in and mention which parts apply to them too.

Once we include other chapters too, we can remove the `\ifgent` around this new `part`, and only wrap it around the MATLAB chapter (it's just that including an empty Part III is a bit strange).